Source/RapyutaSimulationPlugins/Private/Core/RRGameSingleton.cpp

```diff
@@ -97,6 +100,33 @@ bool URRGameSingleton::InitializeResources(bool bInRequestResourceLoading)
     {
         // READ ALL SIM DYNAMIC RESOURCES (UASSETS) INFO FROM DESGINATED [~CONTENT] FOLDERS
         // & REGISTER THEM TO BE ASYNC LOADED INTO [ResourceMap]
+#if (!WITH_EDITOR)
+        // [PAK] --
+        // NOTE: PakLoader is only available in packaged Sim
+        PakLoader = URRUObjectUtils::CreateSelfSubobject<URRPakLoader>(this, TEXT("RRPakLoader"));
+        if (PakLoader->Initialize())
+        {
+            // Load pak files
+            for (const auto& paksBasePath : GetDynamicAssetsBasePathList(ERRResourceDataType::UE_PAK))
```

Comment:
Do we need to load packaged asset?
It causes load all asset in the begining and make start up time slower and slower

I think this is fine as a first step but should stat thinking about loading as demands. we maybe define list of initial loading asset DefaultConfig.ini

Reply:
So this is already added as backlog task to be done.
Basically Sim should load only resources of entity models specified by users, but we need to define configs for them, similar to Import~ModelNameList arg of ImportEntitiesCmdlet.
When users define it, probably in INI, only resource assets (uncooked and Paks) that correspond to stipulated models are loaded.